gulpfile.js

```diff
@@ -770,3 +770,51 @@ gulp.task('release', gulp.series(['build', 'typings', function() {
 
 // The default task builds Blockly.
 gulp.task('default', gulp.series(['build']));
+
+// Stash current state, check out develop, and sync with google/blockly.
+gulp.task('syncDevelop', function(done) {
```

Comment:
Match the rest by using dash rather than camel case. 
Maybe also prefix these tasks with ``git-``, to group them by "feature".